frontend/src/pages/accounts/profile.test.tsx

```diff
@@ -328,7 +326,7 @@ describe("The dashboard", () => {
 
     expect(chromeExtensionBanner).not.toBeInTheDocument();
     expect(firefoxBanner).toBeInTheDocument();
-  });
+  }, 10000);
```

Comment:
Same here - this doesn't look like it should be necessary?

Reply:
https://github.com/mozilla/fx-private-relay/pull/3748#discussion_r1294644891